appservice/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-azureappservice",
-    "version": "0.29.0",
+    "version": "0.29.1",
```

Comment:
This is out of date with master. You'll have to do 0.29.2 if you want to publish this